cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -610,10 +610,22 @@ private void convertEquipmentAtBoundaryNode(Context context, String node) {
     }
 
     private static void convertTwoEquipmentsAtBoundaryNode(Context context, String node, BoundaryEquipment beq1, BoundaryEquipment beq2) {
-        BoundaryLine boundaryLine1 = beq1.createConversion(context).asBoundaryLine(node);
-        BoundaryLine boundaryLine2 = beq2.createConversion(context).asBoundaryLine(node);
+        EquipmentAtBoundaryConversion conversion1 = beq1.createConversion(context);
+        EquipmentAtBoundaryConversion conversion2 = beq2.createConversion(context);
+        BoundaryLine boundaryLine1 = conversion1.asBoundaryLine(node);
+        BoundaryLine boundaryLine2 = conversion2.asBoundaryLine(node);
         if (boundaryLine1 != null && boundaryLine2 != null) {
-            if (boundaryLine2.getId().compareTo(boundaryLine1.getId()) >= 0) {
```

Comment:
Maybe add a comment explaining that we want to manage these configurations in this way on purpose, knowing that the `iidm` model will only be equivalent when only one of the `AcLineSegments` is connected.

Reply:
Done